doc/Possible-key-values.md

```diff
@@ -9,32 +9,21 @@ Key values appear in the following places:
 
 Key values can be any of the following:
 
-- The name of a special key. A complete list of valid special keys follows.
+- The name of a special key. A complete list of valid special keys appears later in this document.
 
-- An arbitrary sequence of characters not containing `:`.
-  This results in a key that writes the specified characters.
+- Any other sequence of characters not containing `:` results in a key that sends those characters.
 
-- The syntax `legend:key_def`.
-  `legend` is the visible legend on the keyboard. It cannot contain `:`.
-  `key_def` can be:
-  + The name of a special key, as listed below.
-  + `'string'` An arbitrary string that can contain `:`. `'` can be added to the string as `` \' ``.
-  + `keyevent:keycode` An Android keycode. They are listed as `KEYCODE_...` in [KeyEvent](https://developer.android.com/reference/android/view/KeyEvent#summary).
+- `keyevent:keycode` An Android keycode: one of the numbers listed with symbol `KEYCODE_...` in the Android [KeyEvent](https://developer.android.com/reference/android/view/KeyEvent#summary) document.
```

Comment:
Keys with a legend are really not the same as keys without a legend. `keyevent:keycode` cannot be without a legend.